src/stores/wallet.ts

```diff
@@ -563,9 +591,8 @@ export default class WalletStore {
       this.selectAccount(id);
     }
     this.observeAccount(id);
-    const res = await this.loadAccount(id);
-    account.importData(parseAccountReponse(res, local));
-    account.loaded = true;
+    const res = await this.fetchAccountData(id);
+    account.importData(parseAccountReponse(res, local), true, true);
```

Comment:
I know that intelliJ shows the argument names, but for the sake of non-intelliJ users it would be nice if the `importData()` function took an options object with named fields as the final parameter, instead of regular boolean arguments. E.g:

```
account.importData(parseAccountReponse(res, local), {
    saveCache: true,
    markAsLoaded: true
});
```

Reply:
Fixed, but an options object for 2 params seems like an overkill.